utils/interpolate.js

```diff
@@ -20,7 +20,7 @@ function interpolate(data, sumKey = 'sum', options, variable, row) {
     const [firstObject] = Object.keys(data) || [];
     const thisYear = get(data, `${firstObject}.dataset`).slice(-4);
 
-    if (thisYear === '2000' || thisYear === '2016') {
+    if (thisYear === '2000' || thisYear === '2017') {
```

Comment:
these feel like things that should all be in config for the application, so that changing these variables does not have to be an app source code change every time. But that can be addressed in some other pass at factfinder when there's more time//less pressing, expansive changes going on :)

Reply:
Agree! There're so many things like this in the API.